content/en/blog/2020/mosn-proxy/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Using MOSN with Istio: an alternative data plane"
+subtitle: "A Cloud Native Proxy for Edge or Service Mesh"
+description: "An alternative sidecar proxy for Istio."
+publishdate: 2020-07-28
+attribution: "Wang Fakang (mosn.io)"
+keywords: [mosn,sidecar,proxy]
+---
+
+[MOSN](https://github.com/mosn/mosn) (Modular Open Smart Network) is a network proxy server written in GoLang. It was built at [Ant Group](https://www.antfin.com) as a sidecar/API Gateway/cloud-native Ingress/Layer 4 or Layer 7 load balancer etc. Over time, we've added extra features, like a multi-protocol framework, multi-process plug-in mechanism, a DSL, and support for the [xDS APIs](https://www.envoyproxy.io/docs/envoy/latest/api-docs/xds_protocol). Supporting xDS means we are now able to use MOSN as the network proxy for Istio.
```

Comment:
add "This configuration is not supported by the Istio project; for help, please see [Learn More](#learn-more) below." at the end of this

(not doing suggested edit as I'm not sure about which e-mail address GitHub will use)

Reply:
Done.